src/utils/getApiStaticPath.ts

```diff
@@ -0,0 +1,35 @@
+import { API_CATEGORIES, API_SUB_CATEGORIES } from '../data/api-categories.mjs';
+import { JS_PLATFORMS } from '@/data/platforms';
+
+type StaticPathType = {
+  params: {
+    platform: string;
+    category: string;
+  };
+};
+
+type ApiStaticPathType = {
+  paths: StaticPathType[];
+  fallback: boolean;
+};
+
+export const getApiStaticPath = (isSubcategory: boolean): ApiStaticPathType => {
```

Comment:
not blocking, but is it possible to get an example of what the paths would look like with the platform and category properties as a comment or something? Trying to understand what the paths should look like